lib/auth/auth.go

```diff
@@ -105,7 +108,9 @@ func NewAuthServer(cfg *InitConfig, opts ...AuthServerOption) *AuthServer {
 		StaticTokens:                  cfg.StaticTokens,
 		ClusterAuthPreference:         cfg.ClusterAuthPreferenceService,
 		UniversalSecondFactorSettings: cfg.UniversalSecondFactorService,
+		trustedClusterBackend:         cfg.TrustedClusterBackend,
```

Comment:
Fixed for `trustedClusterBackend`, it's back to being anonymous.

`developerMode` is private because no one outside the `auth` package uses it? If proxy or node needed it, it's in `service.Config`?